composer.json

```diff
@@ -62,7 +62,10 @@
     "autoload-dev": {
         "psr-0": {
             "": "engine/tests/phpunit/"
-        }
+        },
+		"files": [
+			"engine/tests/phpunit/bootstrap.php"
```

Comment:
How about we leave the duplicate code in TestCase and add cross-referencing comments to keep both in sync? We'll rarely have to touch this and the duplication will be gone in master.